### PR TITLE
[docs] Issue Reports

### PR DESCRIPTION
**Pull request by guifeliper**
Adding some guidelines to create an issue report.

[x] Before opening a pull request, please read the [contributing guidelines](https://github.com/pancakeswap/pancake-frontend/blob/master/CONTRIBUTING.md) first
[x] If your PR is work in progress, open it as `draft`
[x] Before requesting a review, all the checks need to pass
[x] Explain what your PR does

### CODE REVIEW

**Comment by hachiojidev** [comment]
@guifeliper Please fix your commit message before we review this PR.

**Comment by guifeliper** [comment]
> @guifeliper Please fix your commit message before we review this PR.

Sorry, made the commit via GitHub, fixed via `git commit --amend`.

**Review by hachiojidev — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -66,3 +66,45 @@ If a Crowdin translation like this `You have %num% left in your wallet` - would
 ```
 TranslateString(675, `You have ${cakeBalance} left in your wallet`, { num: cakeBalance })
 ```
+
+<a name="bugs"></a>
```
Comment (Chef-Chungus) [inline CONTRIBUTING.md:70]: @guifeliper `name` isn't a valid <a> attribute, and why is this an empty anchor?

**Review by Chef-Chungus — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (guifeliper) [reply CONTRIBUTING.md]: hmm I have followed a template, my first thought was to create a link, but actually does not make sense, you are right.

**Review by Chef-Chungus — APPROVED** [review]

**Pull request merged**